File: Modul 4/Python/PRAK402-2310817310010-AkmallullailSyaban.py
print("Input")
angka = int(input())

print("Output")
for i in range(1, angka+1):
    if i % 2 == 1:
        print(i, end=" ")
print("")
for j in range(angka, 1, -1):
    if j % 2 == 0:
        print(j, end=" ")

File: Modul 4/C/PRAK402-2310817310010-AkmallullailSyaban.c
#include <stdio.h>

int main (){
    int angka;
    printf("Input : ");
    scanf("%d", &angka);
    printf("Output\n");
    for (int i = 1; i <= angka; i++){
        if (i % 2 != 0){
            printf("%d ", i);
        }
    }
    printf("\n");
    for (int i = angka; i >= 1; i--){
        if (i % 2 == 0){
            printf("%d ", i);
        }
    }
    return 0;
}